test.tf

```diff
@@ -213,25 +317,54 @@ resource "checkly_check_group" "test-group1" {
       interval = 5
     }
   }
+
   local_setup_script    = "setup-test"
   local_teardown_script = "teardown-test"
 }
 
 
-# https://github.com/checkly/terraform-provider-checkly/issues/15
-resource "checkly_check_group" "ragog0" {
-  name = "ragog0"
+#----------------------------EXAMPLE----------------------------#
+#--- Adding a check to a check group 
+
+resource "checkly_check" "canonical-api-check-4" {
+  name = "API check 1 belonging to group 1"
+  type = "API"
   activated = true
-  muted = false
-  tags = [
-  "ragog0"
+  muted = true
+  frequency = 720
+  locations = [
+    "eu-central-1",
+    "us-east-2",
   ]
-  concurrency = 3
+  request {
+    method = "GET"
+    url = "https://api.checklyhq.com/public-stats"
+    follow_redirects = true
+  }
+
+  group_id    = checkly_check_group.check-group-1.id
+  group_order = 1            #The `group_order` attribute specifies in which order the checks will be executed: 1, 2, 3, etc.
+
+}
+
+resource "checkly_check" "canonical-api-check-5" {
+  name = "API check 2 belonging to group 1"
+  type = "API"
+  activated = true
+  muted = true
+  frequency = 720
   locations = [
-  "eu-central-1",
-  "eu-west-1",
-  "eu-west-2",
-  "eu-west-3",
-  "eu-north-1",
+    "eu-central-1",
+    "us-east-2",
   ]
-}
\ No newline at end of file
+  request {
+    method = "GET"
+    url = "https://api.checklyhq.com/public-stats"
+    follow_redirects = true
+  }
+
+  group_id    = checkly_check_group.check-group-1.id
+  group_order = 2
+
+}
+
```

Comment:
From the point of view of TDD, I think it's really worth including a piece of test code which demonstrates the fix. For example, if the PR is to fix an error which occurs when the user writes X, we should have a test including X which fails _without_ the commit that fixes the issue, and passes _with_ it. It's important not only to fix the bug, but also to demonstrate the fix, so that it's possible to tell later what it actually was. The test also helps prevent regressions.

Reply:
I think i just did that for example in check-group-1 and check-group-2